### ISSUE TITLE
debian:buster-slim is missing lsb-release package

### ISSUE THREAD

**Issue by kylesloan** [DataDog/ansible-datadog#376]
Hello, when getting datadog agent up and running on the docker image `debian:buster-slim`, we discovered we needed `lsb-release` in order for the role to install without error.  Using datadog release 4.11.0

Without `lsb-release`, ansible does not set the ansible_lsb variable
```
TASK [datadog.datadog : Ensure /etc/apt/trusted.gpg.d/datadog-archive-keyring.gpg exists with same contents as /usr/share/keyrings/datadog-archive-keyring.gpg for older distro versions] **************************
Wednesday 14 July 2021  20:13:42 +0000 (0:00:00.036)       0:00:25.266 ********
fatal: [localhost]: FAILED! => {"msg": "The conditional check '((ansible_distribution == 'Debian' and ansible_lsb.major_release|int < 9) or (ansible_distribution == 'Ubuntu' and ansible_lsb.major_release|int < 16)) and not ansible_check_mode' failed. The error was: error while evaluating conditional (((ansible_distribution == 'Debian' and ansible_lsb.major_release|int < 9) or (ansible_distribution == 'Ubuntu' and ansible_lsb.major_release|int < 16)) and not ansible_check_mode): 'dict object' has no attribute 'major_release'\n\nThe error appears to be in '/mnt/role-mock/.galaxy/roles/datadog.datadog/tasks/pkg-debian.yml': line 45, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: \"Ensure {{ datadog_apt_trusted_d_keyring }} exists with same contents as {{ datadog_apt_usr_share_keyring }} for older distro versions\"\n  ^ here\nWe could be wrong, but this one looks like it might be an issue with\nmissing quotes. Always quote template expression brackets when they\nstart a value. For instance:\n\n    with_items:\n      - {{ foo }}\n\nShould be written as:\n\n    with_items:\n      - \"{{ foo }}\"\n"}
```

**Comment by bkabrda** [comment]
Hey 👋 thanks for the report! I think this should actually be solved by using `ansible_distribution_major_version`. I'll take a closer look and submit a fix for the next version.

**Comment by bkabrda** [comment]
I fixed this in #377, it will be part of the next release. Thanks again for the report!
